### PR TITLE
config: url scheme configuration addition

### PR DESCRIPTION
**Pull request by hachreak**
Signed-off-by: Leonardo Rossi leonardo.r@cern.ch

### CODE REVIEW

File: tests/test_invenio_jsonschemas.py [hunk 1]
```diff
@@ -274,10 +275,18 @@ def test_jsonresolver():
...
+])
+def test_url_mapping(app, dir_factory, url_scheme):
     """Test register schema."""
     app.config['SERVER_NAME'] = 'example.org'
     app.config['JSONSCHEMAS_HOST'] = 'inveniosoftware.org'
+    if url_scheme is not None:
+        app.config['JSONSCHEMAS_URL_SCHEME'] = url_scheme
+    else:
+        # test with default url scheme configuration
+        url_scheme = 'http'
```
Comment (jirikuncar) [inline tests/test_invenio_jsonschemas.py]:
``` python
from invenio_jsonschemas.config import JSONSCHEMAS_URL_SCHEME
url_scheme = JSONSCHEMAS_URL_SCHEME
```

Comment (hachreak) [reply tests/test_invenio_jsonschemas.py]: done

**Comment by hachreak** [comment]
@jirikuncar any idea? :)

```
Warning, treated as error:

/home/travis/build/inveniosoftware/invenio-jsonschemas/docs/api.rst:28: WARNING: autodoc: failed to import module u'invenio_jsonschemas.ext'; the following exception was raised:

Traceback (most recent call last):

  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 519, in import_object

    __import__(self.modname)

ImportError: No module named invenio_jsonschemas.ext
```

**Comment by hachreak** [comment]
@jirikuncar if I invert the order of pytest and sphinx, it works!
I have no idea why because tests clean everything.
Note: the error happens only on travis!

**Comment by jirikuncar** [comment]
@hachreak I suspect the example tests. Can you check if it passes if you remove them?

**Comment by hachreak** [comment]
yes, it's the example app. I already tried. 
And you can see also in travis that current master fails.

**Comment by hachreak** [comment]
@jirikuncar  do you have any clue to resolve the issue about test example app? Because I have a pending PR on CDS that requires this change.
could we simply run before sphinx and then the tests?

**Comment by jirikuncar** [comment]
@hachreak my proposal was to use another virtualenv for example app testing.

**Comment by hachreak** [comment]
what do you mean exactly? O.o
it looks complicated.. XD

**Comment by hachreak** [comment]
@jirikuncar I resolved the problem! :)
could you :shipit:  now?

**Review by jirikuncar — APPROVED** [review]

File: invenio_jsonschemas/config.py [hunk 2]
```diff
@@ -29,3 +29,6 @@
 
 JSONSCHEMAS_ENDPOINT = '/schemas'
 """Default schema endpoint."""
+
+JSONSCHEMAS_URL_SCHEME = 'http'
```
Comment (lnielsen) [inline invenio_jsonschemas/config.py:33]: Should this by default not be `https`?

Comment (hachreak) [reply invenio_jsonschemas/config.py:33]: I set `http` just for back-compatibility.
But, yes, maybe should be `https`

**Pull request merged**